pkg/config/virtio.go

```diff
@@ -663,6 +679,76 @@ func (dev *RosettaShare) FromOptions(options []option) error {
 	return nil
 }
 
+func networkBlockDeviceNewEmpty() *NetworkBlockDevice {
+	return &NetworkBlockDevice{
+		VirtioBlk: VirtioBlk{
+			StorageConfig: StorageConfig{
+				DevName: "nbd",
+			},
+			DeviceIdentifier: "",
+		},
+		Timeout:             time.Duration(15000 * time.Millisecond), // set a default timeout to 15s
```

Comment:
Does this have to be set to a default value? Or can we use the virtualization framework's default if the user did not explicitly set this timeout?